### PR TITLE
#11426: Replace tt_lib in models/experimental/grok,hrnet,lenet,helper…

### PR DESCRIPTION
**Pull request by mouliraj-mcw**
Issue : https://github.com/tenstorrent/tt-metal/issues/11426 Tracking Issue : https://github.com/tenstorrent/tt-metal/issues/11240 , Master Issue : https://github.com/tenstorrent/tt-metal/issues/10532
### Work Done:

- Replace tt_lib usage in `models/experimental/grok`
- Replace tt_lib usage in `models/experimental/hrnet`
- Replace tt_lib usage in `models/experimental/lenet`
- Replace tt_lib usage in `models/helper_funcs.py`

### Checklist
- [x] All Post commit CI 
- [ ] Nightly Fast Dispatch Test
- [ ] Model regression CI testing passes

### CODE REVIEW

File: models/experimental/grok/demo/demo.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 import os
 import torch
 import json
-import tt_lib as ttl
+import ttnn
```
Comment (mcw-anasuya) [inline models/experimental/grok/demo/demo.py:7]: importing ttnn twice

**Comment by tt-rkim** [comment]
Did you run nightly fast dispatch?

**Review by KalaivaniMCW — APPROVED** [review]
Approved. Please run nightly FD and model perf pipelines and make sure there isn't any breaks before merging.

**Review by uaydonat — APPROVED** [review]

**Comment by mouliraj-mcw** [comment]
> Did you run nightly fast dispatch?

yes @tt-rkim . It was Fine.

**Review by mcw-anasuya — APPROVED** [review]

**Pull request merged**